test/unit/local/analysis.js

```diff
@@ -601,6 +602,87 @@ describe('core/local/analysis', function () {
     })
   })
 
+  describe('DirMove(src/ → dst/) + FileUpdate(dst/file)', () => {
+    describe('unlinkDir(src/) + addDir (dst/) + unlink(src/file) + add(dst/file) + change(dst/file)', () => {
+      it('happens when client is running', () => {
```

Comment:
nitpick: the descriptions of the example and the describe around it should be reversed imho, giving something like this:
```javascript
context('when client is running', () => {
  it('translates to unlinkDir(src/) + addDir(dst/) + unlink(src/file) + add(dst/file) + change(dst/file)', () => {
    …
  })
})
```

And the second test would give:
```javascript
context('when client was stopped', () => {
  it('translates to unlinkDir(src/) + addDir(dst/) + unlink(src/file) + add(dst/file, new md5sum) with made up unlink* events', () => {
    …
  })
})

Reply:
The descriptions match the whole file layout, introduced in https://github.com/cozy-labs/cozy-desktop/pull/1354

Reply:
OK, it makes sense.